src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```
@@ -10,6 +10,9 @@
   <PropertyGroup>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsAnyOS)' == 'true'">SR.Process_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
   </PropertyGroup>
+   <PropertyGroup Condition="'$(Platform)' == 'mips64'">
+     <DefineConstants>MIPS64;$(DefineConstants)</DefineConstants>
```

Per the conversion used by others architectures:

```suggestion
     <DefineConstants>TARGET_MIPS64;$(DefineConstants)</DefineConstants>
```

---

Thanks!